### PR TITLE
Add post title to comments view #948

### PR DESCRIPTION
**Pull request by ernestwisniewski**

### CODE REVIEW

File: server/migrations/2020-07-12-100442_add_post_title_to_comments_view/up.sql [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+where closereply.id = cv.id
+;
+
+-- add creator_published to the post view
+drop view post_fast_view;
+drop table post_aggregates_fast;
+drop view post_view;
+drop view post_aggregates_view;
+
+create view post_aggregates_view as
```
Comment (dessalines) [inline server/migrations/2020-07-12-100442_add_post_title_to_comments_view/up.sql:258]: Does post aggregates really need to be re-created? This only alters the comment_view.

File: server/migrations/2020-07-12-100442_add_post_title_to_comments_view/down.sql [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+
+select
+    cav.*,
+    null as user_id,
+    null as my_vote,
+    null as subscribed,
+    null as saved
+from comment_aggregates_fast cav;
+
+--create view user_mention_view as
```
[the remark this answers is not in the document]
Comment (ernestwisniewski) [reply server/migrations/2020-07-12-100442_add_post_title_to_comments_view/down.sql]: Oh, sorry about that.

**Comment by dessalines** [comment]
Better, but looks like you still need to add `post_name` to the reply_view and `user_mention` in rust.

**Comment by dessalines** [comment]
Looks good, merging now with a few changes to the UI.

**Comment by ernestwisniewski** [comment]
> Looks good! Merging now with a few changes to the UI. Thanks for this.

Thanks for help with this one. I find couple things clearer right now. 

At the beginning I wanted to make changes on the front in exactly the same way, but it did not look quite good on the mobile with long post titles, so I moved it to the section below. But I see now that it's not bad at all.

**Pull request merged**